api/proto/info/info.proto

```diff
@@ -284,7 +284,7 @@ message ZInfoDevice {
   string cpuArch = 5;
   string platform = 6;
   uint32 ncpu = 7;
-  uint64 memory = 8;   // in Mbytes
+  uint64 memory = 8;   // Total System memory as see by Eve in Mbytes
```

Comment:
Typo "see".
But not clear what that means. With Xen EVE only "sees" dom0; it asks Xen about the device memory.

So a more accurate comment would be "Total system memory availe (in MBytes). Firmware might eat some memory making this be less than the amount stated by the hardware manufacturer".

I don't know if the Xen hypervisor also decreases this number.

Reply:
Changed it to:

// memory - Total system memory available (in MBytes). Firmware might use
  // some memory making it unavailable to the hypervisor - So this could be
  // less than the amount stated by the hardware manufacturer